From 0ace1e80552a40ebe92e4298340263fd9bce0bb3 Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Mon, 30 Mar 2020 16:57:45 +0800
Subject: [PATCH 04/22] arch: arm64: dts: fix some dts build warnings
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

detail fixing as below:
- replace “6A” with “6a”
- replace “50” with “51”
- use the common string “spi” as the name of spi dts node

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi          |  6 +++---
 arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi          |  8 ++++----
 arch/arm64/boot/dts/freescale/fsl-imx8mn.dtsi          |  8 ++++----
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi          | 10 +++++-----
 arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi      |  8 ++++----
 arch/arm64/boot/dts/freescale/fsl-imx8qm-ddr4-arm2.dts |  8 ++++----
 arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi   |  6 +++---
 .../freescale/fsl-imx8qm-lpddr4-arm2-lpspi-slave.dts   |  3 ++-
 .../boot/dts/freescale/fsl-imx8qm-mek-dom0-dpu2.dts    |  2 +-
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts  |  2 +-
 .../boot/dts/freescale/fsl-imx8qm-mek-domu-car.dts     |  4 ++--
 .../boot/dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts    |  4 ++--
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts  |  4 ++--
 .../arm64/boot/dts/freescale/fsl-imx8qm-mek-ov5640.dts |  4 ++--
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi      | 10 +++++-----
 .../freescale/fsl-imx8qxp-lpddr4-arm2-lpspi-slave.dts  |  3 ++-
 .../boot/dts/freescale/fsl-imx8qxp-mek-ov5640.dts      |  2 +-
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi     |  4 ++--
 arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi      |  4 ++--
 19 files changed, 51 insertions(+), 49 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
index 1c562d3a81c5..9b8c957017af 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
@@ -2742,7 +2742,7 @@
 		interrupts = <GIC_SPI 131 IRQ_TYPE_LEVEL_HIGH>;
 	};
 
-	lpspi0: lpspi@5a000000 {
+	lpspi0: spi@5a000000 {
 		compatible = "fsl,imx7ulp-spi";
 		reg = <0x0 0x5a000000 0x0 0x10000>;
 		interrupts = <GIC_SPI 216 IRQ_TYPE_LEVEL_HIGH>;
@@ -2758,7 +2758,7 @@
 		status = "disabled";
 	};
 
-	lpspi2: lpspi@5a020000 {
+	lpspi2: spi@5a020000 {
 		compatible = "fsl,imx7ulp-spi";
 		reg = <0x0 0x5a020000 0x0 0x10000>;
 		interrupts = <GIC_SPI 218 IRQ_TYPE_LEVEL_HIGH>;
@@ -3478,7 +3478,7 @@
 		status = "disabled";
 	};
 
-	flexspi0: flexspi@5d120000 {
+	flexspi0: spi@5d120000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8qxp-flexspi";
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
index 3394ec323ced..012903f21302 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
@@ -958,7 +958,7 @@
 		status = "disabled";
 	};
 
-	flexspi: flexspi@30bb0000 {
+	flexspi: spi@30bb0000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mm-flexspi";
@@ -973,7 +973,7 @@
 		status = "disabled";
 	};
 
-	ecspi1: ecspi@30820000 {
+	ecspi1: spi@30820000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mm-ecspi", "fsl,imx51-ecspi";
@@ -987,7 +987,7 @@
 		status = "disabled";
 	};
 
-	ecspi2: ecspi@30830000 {
+	ecspi2: spi@30830000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mm-ecspi", "fsl,imx51-ecspi";
@@ -1001,7 +1001,7 @@
 		status = "disabled";
 	};
 
-	ecspi3: ecspi@30840000 {
+	ecspi3: spi@30840000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mm-ecspi", "fsl,imx51-ecspi";
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mn.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mn.dtsi
index f68c8913a6c5..ed61d20ad50d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mn.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mn.dtsi
@@ -646,7 +646,7 @@
 			     <GIC_SPI 48 IRQ_TYPE_LEVEL_HIGH>;
 	};
 
-	ecspi1: ecspi@30820000 {
+	ecspi1: spi@30820000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mm-ecspi", "fsl,imx51-ecspi";
@@ -660,7 +660,7 @@
 		status = "disabled";
 	};
 
-	ecspi2: ecspi@30830000 {
+	ecspi2: spi@30830000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mm-ecspi", "fsl,imx51-ecspi";
@@ -674,7 +674,7 @@
 		status = "disabled";
 	};
 
-	ecspi3: ecspi@30840000 {
+	ecspi3: spi@30840000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mm-ecspi", "fsl,imx51-ecspi";
@@ -891,7 +891,7 @@
 		status = "disabled";
 	};
 
-	flexspi: flexspi@30bb0000 {
+	flexspi: spi@30bb0000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mm-flexspi";
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 840772922043..797fba550524 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -791,7 +791,7 @@
 		status = "disabled";
 	};
 
-	ecspi1: ecspi@30820000 {
+	ecspi1: spi@30820000 {
 		compatible = "fsl,imx6ul-ecspi", "fsl,imx51-ecspi";
 		reg = <0x0 0x30820000 0x0 0x10000>;
 		interrupts = <GIC_SPI 31 IRQ_TYPE_LEVEL_HIGH>;
@@ -801,7 +801,7 @@
 		status = "disabled";
 	};
 
-	ecspi1: ecspi@30820000 {
+	ecspi1: spi@30820000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mq-ecspi", "fsl,imx51-ecspi";
@@ -813,7 +813,7 @@
 		status = "disabled";
 	};
 
-	ecspi2: ecspi@30830000 {
+	ecspi2: spi@30830000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mq-ecspi", "fsl,imx51-ecspi";
@@ -825,7 +825,7 @@
 		status = "disabled";
 	};
 
-	ecspi3: ecspi@30840000 {
+	ecspi3: spi@30840000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mq-ecspi", "fsl,imx51-ecspi";
@@ -1204,7 +1204,7 @@
 		only-dma-mask32 = <1>;
 	};
 
-	qspi: qspi@30bb0000 {
+	qspi: spi@30bb0000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx7d-qspi";
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi
index e59804e0d8b1..4f9c66a85400 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi
@@ -708,9 +708,9 @@
 	clock-frequency = <100000>;
 	status = "okay";
 
-	max9286_mipi@6A	 {
+	max9286_mipi@6a	 {
 		compatible = "maxim,max9286_mipi";
-		reg = <0x6A>;
+		reg = <0x6a>;
 		clocks = <&clk IMX8QM_CLK_DUMMY>;
 		clock-names = "capture_mclk";
 		mclk = <27000000>;
@@ -733,9 +733,9 @@
 	clock-frequency = <100000>;
 	status = "disabled";
 
-	max9286_mipi@6A	 {
+	max9286_mipi@6a	 {
 		compatible = "maxim,max9286_mipi";
-		reg = <0x6A>;
+		reg = <0x6a>;
 		clocks = <&clk IMX8QM_CLK_DUMMY>;
 		clock-names = "capture_mclk";
 		mclk = <27000000>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-ddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-ddr4-arm2.dts
index d49de6a68cf0..a5ee5fa8b6a1 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-ddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-ddr4-arm2.dts
@@ -679,9 +679,9 @@
 	clock-frequency = <100000>;
 	status = "okay";
 
-	max9286_mipi@6A	 {
+	max9286_mipi@6a	 {
 		compatible = "maxim,max9286_mipi";
-		reg = <0x6A>;
+		reg = <0x6a>;
 		clocks = <&clk IMX8QM_CLK_DUMMY>;
 		clock-names = "capture_mclk";
 		mclk = <27000000>;
@@ -704,9 +704,9 @@
 	clock-frequency = <100000>;
 	status = "disabled";
 
-	max9286_mipi@6A	 {
+	max9286_mipi@6a	 {
 		compatible = "maxim,max9286_mipi";
-		reg = <0x6A>;
+		reg = <0x6a>;
 		clocks = <&clk IMX8QM_CLK_DUMMY>;
 		clock-names = "capture_mclk";
 		mclk = <27000000>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index 01236e4c618b..d51dc073b5c6 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -2915,7 +2915,7 @@
 		status = "disabled";
 	};
 
-	lpspi0: lpspi@5a000000 {
+	lpspi0: spi@5a000000 {
 		compatible = "fsl,imx7ulp-spi";
 		reg = <0x0 0x5a000000 0x0 0x10000>;
 		interrupts = <GIC_SPI 216 IRQ_TYPE_LEVEL_HIGH>;
@@ -2931,7 +2931,7 @@
 		status = "disabled";
 	};
 
-	lpspi3: lpspi@5a030000 {
+	lpspi3: spi@5a030000 {
 		compatible = "fsl,imx7ulp-spi";
 		reg = <0x0 0x5a030000 0x0 0x10000>;
 		interrupts = <GIC_SPI 219 IRQ_TYPE_LEVEL_HIGH>;
@@ -4074,7 +4074,7 @@
 		status = "disabled";
 	};
 
-	flexspi0: flexspi@5d120000 {
+	flexspi0: spi@5d120000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8qm-flexspi";
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-lpspi-slave.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-lpspi-slave.dts
index c38b3970bb8d..f416a876415b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-lpspi-slave.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-lpspi-slave.dts
@@ -14,7 +14,7 @@
 
 #include "fsl-imx8qm-lpddr4-arm2-lpspi.dts"
 
-/delete-node/&spidev0;
+/delete-node/ &spidev0;
 
 &pinctrl_lpspi3 {
 	fsl,pins = <
@@ -26,6 +26,7 @@
 };
 
 &lpspi3 {
+	#address-cells = <0>;
 	pinctrl-0 = <&pinctrl_lpspi3>;
 	/delete-property/ cs-gpios;
 	spi-slave;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0-dpu2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0-dpu2.dts
index 0b2e54acefd5..2f6bfa8e37cd 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0-dpu2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0-dpu2.dts
@@ -644,7 +644,7 @@
 		xen,passthrough;
 	};
 
-	typec_ptn5110: typec@50 {
+	typec_ptn5110: typec@51 {
 		xen,passthrough;
 	};
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
index 38e92b9da2df..d9929476d284 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
@@ -774,7 +774,7 @@
 		xen,passthrough;
 	};
 
-	typec_ptn5110: typec@50 {
+	typec_ptn5110: typec@51 {
 		xen,passthrough;
 	};
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-car.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-car.dts
index d8ff2c05bbc3..3b2601d458cc 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-car.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-car.dts
@@ -567,9 +567,9 @@
 		status = "okay";
 		clock-frequency = <1000000>;
 		/*pinctrl-0 = <&pinctrl_mipi_csi0_en>;*/
-		max9286_mipi@6A	 {
+		max9286_mipi@6a	 {
 			compatible = "maxim,max9286_mipi";
-			reg = <0x6A>;
+			reg = <0x6a>;
 			clocks = <&clk IMX8QM_CLK_DUMMY>;
 			clock-names = "capture_mclk";
 			mclk = <27000000>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts
index 23c56c263594..3da395245be9 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts
@@ -978,9 +978,9 @@
 	status = "okay";
 
 
-	max9286_mipi@6A	 {
+	max9286_mipi@6a	 {
 		compatible = "maxim,max9286_mipi";
-		reg = <0x6A>;
+		reg = <0x6a>;
 		clocks = <&clk IMX8QM_CLK_DUMMY>;
 		clock-names = "capture_mclk";
 		mclk = <27000000>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
index e7bffc31bcb5..4378950fb3d4 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
@@ -1225,9 +1225,9 @@
 	status = "okay";
 
 
-	max9286_mipi@6A	 {
+	max9286_mipi@6a	 {
 		compatible = "maxim,max9286_mipi";
-		reg = <0x6A>;
+		reg = <0x6a>;
 		clocks = <&clk IMX8QM_CLK_DUMMY>;
 		clock-names = "capture_mclk";
 		mclk = <27000000>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-ov5640.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-ov5640.dts
index 58f3c16738aa..05e4d28cb42f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-ov5640.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-ov5640.dts
@@ -91,7 +91,7 @@
 		};
 	};
 
-	/delete-node/max9286_mipi@6A;
+	/delete-node/max9286_mipi@6a;
 };
 
 &i2c0_mipi_csi1 {
@@ -119,7 +119,7 @@
 		};
 	};
 
-	/delete-node/max9286_mipi@6A;
+	/delete-node/max9286_mipi@6a;
 };
 
 &mipi_csi_0 {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
index f939e06f3f83..c59e07f7cc36 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
@@ -857,7 +857,7 @@
 		interrupt-open-drain;
 	};
 
-	typec_ptn5110: typec@50 {
+	typec_ptn5110: typec@51 {
 		compatible = "nxp,ptn5110";
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_typec>;
@@ -1192,9 +1192,9 @@
 	clock-frequency = <100000>;
 	status = "okay";
 
-	max9286_mipi@6A	 {
+	max9286_mipi@6a	 {
 		compatible = "maxim,max9286_mipi";
-		reg = <0x6A>;
+		reg = <0x6a>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_mipi_csi0_en_rst>;
 		clocks = <&clk IMX8QM_CLK_DUMMY>;
@@ -1218,9 +1218,9 @@
 	clock-frequency = <100000>;
 	status = "okay";
 
-	max9286_mipi@6A	 {
+	max9286_mipi@6a	 {
 		compatible = "maxim,max9286_mipi";
-		reg = <0x6A>;
+		reg = <0x6a>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_mipi_csi1_en_rst>;
 		clocks = <&clk IMX8QM_CLK_DUMMY>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-lpspi-slave.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-lpspi-slave.dts
index 8c2661370de4..33d24a753130 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-lpspi-slave.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-lpspi-slave.dts
@@ -14,7 +14,7 @@
 
 #include "fsl-imx8qxp-lpddr4-arm2-lpspi.dts"
 
-/delete-node/&spidev0;
+/delete-node/ &spidev0;
 
 &pinctrl_lpspi2 {
 	fsl,pins = <
@@ -26,6 +26,7 @@
 };
 
 &lpspi2 {
+	#address-cells = <0>;
 	pinctrl-0 = <&pinctrl_lpspi2>;
 	/delete-property/ cs-gpios;
 	spi-slave;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-ov5640.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-ov5640.dts
index 4bd68f2ae12a..bad039438b61 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-ov5640.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-ov5640.dts
@@ -109,7 +109,7 @@
 		};
 	};
 
-	/delete-node/max9286_mipi@6A;
+	/delete-node/max9286_mipi@6a;
 };
 
 &mipi_csi_0 {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
index 7b33c47be66c..77ae9f6ef6af 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
@@ -1152,9 +1152,9 @@
 	clock-frequency = <100000>;
 	status = "okay";
 
-	max9286_mipi@6A	 {
+	max9286_mipi@6a	 {
 		compatible = "maxim,max9286_mipi";
-		reg = <0x6A>;
+		reg = <0x6a>;
 		clocks = <&clk IMX8QXP_CLK_DUMMY>;
 		clock-names = "capture_mclk";
 		mclk = <27000000>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi
index 0d155c55afb6..702c0c714ef2 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi
@@ -606,9 +606,9 @@
 		status = "okay";
 	};
 
-	max9286_mipi@6A	 {
+	max9286_mipi@6a	 {
 		compatible = "maxim,max9286_mipi";
-		reg = <0x6A>;
+		reg = <0x6a>;
 		clocks = <&clk IMX8QXP_CLK_DUMMY>;
 		clock-names = "capture_mclk";
 		mclk = <27000000>;
-- 
2.17.1

